hieradata_aws/class/integration/jenkins.yaml

```diff
@@ -5,6 +5,24 @@ govuk_jenkins::config::theme_colour: '#005EA5'
 govuk_jenkins::config::theme_text_colour: 'white'
 govuk_jenkins::config::theme_environment_name: 'AWS Integration'
 
+govuk_jenkings::config::user_permissions:
+  -
+    user: 'jenkins_api_user'
+    permissions: *jenkins_admin_permission_list
+  -
+    user: 'alphagov*GOV.UK'
+    permissions: *jenkins_admin_permission_list
+  -
+    user: 'anonymous'
+    permissions:
+      - 'hudson.model.Hudson.Read'
+      - 'hudson.model.Item.Discover'
+  -
+    user: 'github'
+    permissions:
+      - 'hudson.model.Item.Build'
+      - 'hudson.model.Item.Read'
```

Comment:
Is adding this here going to change anything? It looks to me that the same configuration is applied at the class level for the Jenkins machines in AWS: https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/class/jenkins.yaml#L35-L51

Reply:
Ah, I see now, you're adding this here to keep the existing behaviour for Integration, while changing the behaviour for Staging and Production...